my-new-webapp-servlet/src/main/webapp/WEB-INF/web.xml

```diff
@@ -5,20 +5,37 @@
          version="4.0">
     <display-name>MyWebApp</display-name>
 
+    <context-param>
+        <param-name>log4j-config-location</param-name>
+        <param-value>WEB-INF/log4j.properties</param-value>
+    </context-param>
+
     <filter>
         <filter-name>SimpleCORSFilter</filter-name>
         <filter-class>servlet.filter.SimpleCORSFilter</filter-class>
     </filter>
+    <filter>
+        <filter-name>AuthFilter</filter-name>
+        <filter-class>servlet.filter.AuthFilter</filter-class>
+    </filter>
+    <filter>
+        <filter-name>AccessFilter</filter-name>
+        <filter-class>servlet.filter.AccessFilter</filter-class>
+    </filter>
```

Comment:
It is not required to register nor servlets nor filters neither listeners if you mark them with annotations like `@WebServlet`, `@WebFilter` etc. So please choose one particular method to register web resources, not mix them.